packages/ckeditor5-paste-from-office/tests/filters/list.js

```diff
@@ -104,6 +107,38 @@ describe( 'PasteFromOffice - filters', () => {
 				);
 			} );
 
+			it( 'handles "legal-list" when multi-level-list is loaded', () => {
+				const level1 = 'style="mso-list:l0 level1 lfo0"';
+				const styles = '@list l0:level1\n' +
+					'{mso-level-text:"%1\\.%2\\.";}';
```

Comment:
This is not verifying the regexp. I'd suggest adding more cases or adding a payload-based test.